.github/workflows/forward_model_load_check.yml

```
@@ -0,0 +1,78 @@
+name: Model checks
+
+# This file is for checking that models generated from newer VW code do not cause crashes when loaded into older VW code
+# Generate a model using latest wheel and load the same model using the latest vw python release
+
+on:
+  push:
+    branches:
+      - master
+      - 'releases/**'
+  pull_request:
+    branches:
+      - '*'
+
+jobs:
+  python-build:
+    name: Build latest python wheel
+    container:
+      image: vowpalwabbit/manylinux2010-build:latest
+    runs-on: ubuntu-latest
+    steps:
+      # v1 must be used because newer versions require a node.js version that will not run on this old image.
+      - uses: actions/checkout@v1
+        with:
+          submodules: recursive
+      - name: Build wheel
+        shell: bash
+        run: |
+          /opt/python/cp310-cp310/bin/pip wheel . -w wheel_output/ --global-option --cmake-options="-DSTATIC_LINK_VW_JAVA=On;-DPython_INCLUDE_DIR='/opt/python/cp310-cp310/include/python3.10/'" --verbose
+          auditwheel repair wheel_output/*whl -w audit_output/
+      - name: Upload built wheel
+        uses: actions/upload-artifact@v1
+        with:
+          name: test_manylinux_amd64_3.10
+          path: audit_output/
+  generate-latest-model:
+    name: Generate model using latest artifact
+    needs: python-build
+    container:
+      image: python:3.10
+    runs-on: ubuntu-latest
+    steps:
+      # v1 must be used because newer versions require a node.js version that will not run on this old image.
+      - uses: actions/checkout@v1
+      - name: Download artifact
+        uses: actions/download-artifact@v1
+        with:
```

I think it would be best to avoid using a docker image if we can rely on the setup action. I think it would be faster.
```suggestion
    name: Generate model using latest artifact
    needs: python-build
    runs-on: ubuntu-latest
    steps:
      - uses: actions/checkout@v3
      - uses: actions/setup-python@v4
        with:
          python-version: '3.10' 
      - name: Download artifact
        uses: actions/download-artifact@v1
        with:
```

---

cool